styles/landing/landing.scss

```diff
@@ -1,5 +1,28 @@
 // Rest -------------------------------------------------------------------- //
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,700,900");
+/* roboto-300 - vietnamese_latin-ext_latin_greek-ext_greek_cyrillic-ext_cyrillic */
+@font-face {
+	font-family: "Roboto";
+	font-style: normal;
+	font-weight: 300;
+	src: local(""), url("../../../fonts/roboto-v29-300.woff2") format("woff2");
```

Comment:
Just curious -- why does this need `"../../../"`? What directory are we in when these need to be loaded? I can see that it's correct from the point of view of the Lychee front end sources, but once those are compiled, doesn't that whole nested folder structure get forgotten?